src/components/mastheadOptions/mastheadOptions.js

```diff
@@ -42,8 +41,8 @@ MastheadOptions.propTypes = {
 };
 
 MastheadOptions.defaultProps = {
-  logoutUser: helpers.noop,
-  showAboutModal: helpers.noop
+  logoutUser: null,
+  showAboutModal: null
```

Comment:
Why change to null?

Reply:
Could also take the checks out and go back towards just using `helpers.noop` ... either way does work